Doc/glossary.rst

```diff
@@ -1280,6 +1280,13 @@ Glossary
       and ending with double underscores.  Special methods are documented in
       :ref:`specialnames`.
 
+   standard library
+      The collection of :term:`packages <package>`, :term:`modules <module>`
+      and :term:`extension modules <extension module>` distributed as a part
+      of the official Python interpreter package, or an individual member of
+      that collection.  The exact membership of the collection may vary based
```

Comment:
As a noun, 'standard library' refers to the entire collecton.  As an adjective, it can be applied to any member.  Giving both definitions in one sentence is confusing.
```suggestion
      The collection of :term:`packages <package>`, :term:`modules <module>`
      and :term:`extension modules <extension module>` distributed as a part
      of the official Python interpreter package.  As an adjective, "an member of"
      that collection.  The exact membership of the collection may vary based
```